tests/bud.bats

```diff
@@ -531,6 +532,7 @@ _EOF
 
   cat > $contextdir/Dockerfile << _EOF
 FROM alpine
+ARG key=value
```

Comment:
Do we still have cache test coverage for cases where there isn't an `ARG` before the heredoc?